http/src/commonTest/kotlin/com/mirego/trikot/http/RequestBuilderTest.kt

```diff
@@ -32,14 +32,24 @@ class RequestBuilderTest {
     }
 
     @Test
-    fun `given empty baseUrl and path when buildUrl then return empty string`() {
+    fun testGivenEmptyBaseUrlWhenBuildUrlThenReturnPath() {
+
```

Comment:
@thermech `ktlint` doesn’t raise an error on this blank line? 🤔

Reply:
Double blank line only